todo_app/app.py

```diff
@@ -1,14 +1,38 @@
-from flask import Flask
-
+from flask import Flask, request, render_template, redirect, url_for
+import json
 from todo_app.flask_config import Config
+from todo_app.data import session_items as session
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=["GET", "POST"])
 def index():
-    return 'Hello World!'
+    error_message = None
+    if request.method == "POST":
```

Comment:
The fact we handle this under the same route makes this a little more complex; splitting out adding cards into its own route might make more sense (with the endpoint potentially just returning an error if the problem occurs)

Reply:
Cheers Jack, done.